Sources/PostgresNIO/Data/PostgresDataType.swift

```diff
@@ -227,3 +222,13 @@ public struct PostgresDataType: Codable, Equatable, ExpressibleByIntegerLiteral,
         return self.knownSQLName ?? "UNKNOWN \(self.rawValue)"
     }
 }
+
+// TODO: The Codable conformance does not make any sense. Let's remove this with next major break.
```

Comment:
Was there a point in early Swift 5 before `RawRepresentable` automatically conferred `Codable`? That would make sense of why the redundant conformance was left in. (I mean heck, I tend to write out `Equatable, Hashable` by habit and then grumble at the compiler warning complaining `Equatable` is redundant...)

Though, given that the conformance is automatically bestowed regardless, does the removal of the explicit declaration actually have to wait for a major version break?

(Hm, do structs get it automatically? Or just enums? Either way, the enum in the other PR could lose it without a major version break at least.)